website/pages/style.css

```diff
@@ -82,6 +95,225 @@
   color: white;
 }
 
+/* Print Styles - Hide Elements */
+@media print {
+  iframe,
+  .g-mega-nav,
+  .g-product-subnav,
+  .g-footer,
+  [aria-hidden='true'],
+  [id='sidebar'],
+  [id='edit-this-page'],
+  [id='jump-to-section'],
+  [id='__next-build-watcher'] {
+    display: none !important;
+  }
+}
+
+/* Print Styles - Page Spacing */
+@media print {
+  @page {
+    margin: 2cm 0.5cm;
+  }
+
+  @page :first {
+    margin-top: 0;
+  }
+
+  @page :last {
+    margin-top: 0;
+  }
+
+  blockquote {
+    break-inside: avoid;
+  }
+
+  body {
+    margin-bottom: 2cm;
+    margin-top: 2cm;
+  }
+
+  dl,
+  ol,
+  ul {
+    break-before: avoid;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    break-after: avoid;
+    break-inside: avoid;
+  }
+
+  img {
+    break-inside: avoid;
+    break-after: avoid;
+  }
+
+  pre,
+  table {
+    break-inside: avoid;
+  }
+
+  pre {
+    white-space: pre-wrap;
+  }
+}
+
+/* Print Styles - Override @hashicorp/react-global-styles/global.css */
```

Comment:
What do you think about putting this and the `.hljs` styles as overrides both under https://github.com/hashicorp/web-components/blob/master/packages/global-styles/code-highlighting.css